### PR TITLE
LUN-269: Feedback for a word logic

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

**Review by steffenkleinle — COMMENTED** [review]
Tested on android :+1: When is it possible to give feedback for a discipline?
image

**Review by steffenkleinle — CHANGES_REQUESTED** [review]
Suggest alternative word does not work

**Comment by SteffiStuffel** [comment]
> Tested on android +1 When is it possible to give feedback for a discipline? image

Currently it is only possible via the api, but not in the app. No designs or feature request for this yet.

File: src/routes/vocabulary-detail/components/AlternativeWordsSection.tsx [hunk 1]
```diff
@@ -51,11 +51,10 @@ const AlternativeWordsSection = ({ document }: Props): JSX.Element => (
         </>
       )}
 
-      {/* Will be implemented in LUN-269 */}
-      <SuggestionContainer>
+      <SuggestionPressable>
```
[the remark this answers is not in the document]
Comment (SteffiStuffel) [reply src/routes/vocabulary-detail/components/AlternativeWordsSection.tsx:56]: Fixed, suggest alternative should work now.

File: src/components/ExerciseHeader.tsx [hunk 2]
```diff
@@ -29,14 +30,16 @@ const HeaderRightContainer = styled.View`
 const ProgressText = styled(ContentSecondary)`
   margin-right: ${props => props.theme.spacings.sm};
 `
-// TODO Remove comment when LUN-269 is ready
-// const MenuIconPrimary = styled(MenuIcon)`
-//   color: ${props => props.theme.colors.primary};
-// `
+
+const MenuIconPrimary = styled(MenuIcon)`
```
[the remark this answers is not in the document]
Comment (SteffiStuffel) [reply src/components/ExerciseHeader.tsx:34]: Primary is our theme color, it is also named like this.

File: src/components/FeedbackModal.tsx [hunk 3]
```diff
@@ -25,8 +35,7 @@ const FeedbackModal: React.FC<FeedbackModalProps> = ({ visible, onClose }: Feedb
     onClose()
   }
   const onSubmit = (): void => {
-    // TODO Submit Feedback LUN-269
-    onCloseFeedback()
+    sendFeedback(`${message} ${email}`, feedbackType, feedbackForId).catch(reportError).finally(onCloseFeedback)
```
[the remark this answers is not in the document]
Comment (SteffiStuffel) [reply src/components/FeedbackModal.tsx:38]: no there isn't

**Comment by SteffiStuffel** [comment]
> Suggest alternative word does not work

Fixed it.

**Review by steffenkleinle — APPROVED** [review]
Not tested again

**Pull request merged**